<launch>
    <arg name="arduino_baudrate" default="57600"/>
    <arg name="arduino_port" default="/dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0"/>

    <arg name="led_baudrate" default="19200"/>
    <arg name="led_device_port" default="/dev/serial/by-id/usb-Silicon_Labs_CP2102_USB_to_UART_Bridge_Controller_0001-if00-port0"/>

    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
        <param name="port" value="$(arg arduino_port)"/>
        <param name="baud" value="$(arg arduino_baudrate)"/>
    </node>

  <node pkg="ros_farmer_pc" type="led_device.py" name="led_device">
    <param name="_led_port" value="$(arg led_device_port)"/>
    <param name="_led_baudrate" value="$(arg led_baudrate)"/>
  </node>
</launch>